packages/xo-web/src/common/xo/index.js

```diff
@@ -899,6 +899,9 @@ export const isHyperThreadingEnabledHost = host =>
     id: resolveId(host),
   })
 
+export const installCertificateOnHost = (id, props) =>
```

Comment:
`host` or `hostId` instead of `id` to be more clear.

Reply:
There is a `xo-server` API which makes clear what should be passed.